Sources/RInAppMessaging/Protocols/RemoteNotificationRequestable.swift

```diff
@@ -0,0 +1,16 @@
+import UserNotifications
+import class UIKit.UIApplication
+
+protocol RemoteNotificationRequestable {
+    func requestAuthorization(options: UNAuthorizationOptions,
+                              completionHandler: @escaping (Bool, Error?) -> Void)
+    func registerForRemoteNotifications()
+}
+
+extension RemoteNotificationRequestable {
+    func registerForRemoteNotifications() {
+        UIApplication.shared.registerForRemoteNotifications()
```

Comment:
Interesting. The RPushPNP SDK does not register to APNS. Customers have to do it in their apps.

Reply:
Yeah. It is supposed to be a hands-free feature.